backend/dal/Repositories/ActivityRepository.cs

```diff
@@ -87,22 +90,26 @@ public PimsActivityInstance Add(PimsActivityInstance instance)
         public PimsActivityInstance Update(PimsActivityInstance instance)
         {
             instance.ThrowIfNull(nameof(instance));
-            GetById(instance.ActivityInstanceId);
-            if (instance.ActivityTemplate != null)
```

Comment:
rather then individually blacklisting all of the related properties that shouldn't be updated (which is error prone as it is easy to forget to add a new property here). Decided to change this to a whitelist model instead, where by default only the base entity is updated.